jp2_pc/Source/Tools/CollisionEditor/CollisionEditorDlg.cpp

```diff
@@ -5,11 +5,12 @@
 
 #include "CollisionEditor.h"
 #include "CollisionEditorDlg.h"
-#include "bstring.h"
-#include "set.h"
+#include <string>
+#include <set>
 #include "Database.h"
 #include "EditMaterialDlg.h"
 #include "SampleListDlg.h"
+#include "resource.h"
```

Comment:
You've already included this in `CollisionEditorDlg.h`. Possible oversight?

Reply:
Yes, that was redundant. Fixed.